ethernet/EthMacCore/RoCEv2/README.md

```diff
@@ -0,0 +1 @@
+@FilMarini: Please write up some text about how these modules were auto-generated from BlueRdma
```

Comment:
I think it would make sense to change the path of this folder. I dont see the reason why the `RoCEv2` folder should stay inside the `EthMacCore` folder.
Maybe we could only leave the `blue-crc` folder in here, boit the RoCE engine I think should be in the `ethernet` folder

Reply:
https://github.com/slaclab/surf/commit/146c7f65bb3b3e7fd1dcadf90a931a6b8e8b4b15